woocommerce/wc-admin
# Admin Notes: Add CRUD and Data Stores

See #176 

Ready for testing.

This PR implements the data stores for the admin notes. It does not include the REST API interfaces (that will be in a subsequent PR) as this PR is already too big. Unit tests will also be provided in a subsequent PR once the unit test PR has landed - in the mean time, to test, you'll also need this plugin: https://github.com/Automattic/wc-admin-notes-test-bench

To test:
- Install and activate wc-admin using this branch
- Install and activate the test bench plugin
- Go to wp-admin > Tools > WC Admin Notes Test Bench
- Add a new note, e.g.

<img width="653" alt="screen shot 2018-09-26 at 10 44 54 am" src="https://user-images.githubusercontent.com/1595739/46098439-3e95ac00-c179-11e8-957c-8d26eb79c9fc.png">

- After adding, verify it appears in the test bench, e.g.:

<img width="1396" alt="screen shot 2018-09-26 at 10 46 03 am" src="https://user-images.githubusercontent.com/1595739/46098499-6553e280-c179-11e8-8df8-4dce4dc8e70b.png">

- You may also verify the note and action in SQL, e.g.:

<img width="1338" alt="screen shot 2018-09-26 at 10 47 07 am" src="https://user-images.githubusercontent.com/1595739/46098545-887e9200-c179-11e8-87b3-175d1110a390.png">

Enjoy!
